template.json

```diff
@@ -817,6 +817,17 @@
                     "initiation": 0
                 }
             },
+            "technomancer": {
+                "technomancer": {
+                    "attribute": "willpower",
+                    "fade": {
```

Comment:
What's your intention with this fade ModifiableValue field?

Reply:
Nothing yet, I just saw that there was an issue to make it modifiable and magic has a similiar field

Reply:
Then I'll remove this.

Fade tests will be modifable by ActiveEffects in the future. I'll also check the magic one, if it's even in use anymore and remove it if not. :)